### PR TITLE
new: add refreshID in pingresults

### PR DESCRIPTION
**Pull request by sibicramesh**

### CODE REVIEW

**Comment by sibicramesh** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "pass-refreshid",
    "component": "backend",
    "pr-id": "771",
    "commit-sha": "c53f3901052008eefd26b59d515ccff54de87ef6",
    "pipeline": "master"
  },
  {
    "project": "pass-refreshid",
    "component": "gaia",
    "pr-id": "509",
    "commit-sha": "5b1a5f2c6253d84ffc7c74cecb1f692eec83d13d",
    "pipeline": "master"
  },
  {
    "project": "pass-refreshid",
    "component": "enforcerd",
    "pr-id": "1769",
    "commit-sha": "8ed276ce885578c4eddd7183e1e8a1f7955c835f",
    "pipeline": "master"
  }
]
```

File: specs/pingrequest.spec [hunk 1]
```diff
@@ -24,3 +24,8 @@ attributes:
     exposed: true
     read_only: true
     autogenerated: true
+
+  - name: refreshID
```
Comment (primalmotion) [inline specs/pingrequest.spec:28]: Don't we want this required?

**Comment by sibicramesh** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "pass-refreshid",
    "component": "backend",
    "pr-id": "771",
    "commit-sha": "c53f3901052008eefd26b59d515ccff54de87ef6",
    "pipeline": "master"
  },
  {
    "project": "pass-refreshid",
    "component": "gaia",
    "pr-id": "509",
    "commit-sha": "81d9ed7e63a79f6824cfed148b666bf7678f45e3",
    "pipeline": "master"
  },
  {
    "project": "pass-refreshid",
    "component": "enforcerd",
    "pr-id": "1769",
    "commit-sha": "8ed276ce885578c4eddd7183e1e8a1f7955c835f",
    "pipeline": "master"
  }
]
```

**Comment by sibicramesh** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "pass-refreshid",
    "component": "gaia",
    "pr-id": "509",
    "commit-sha": "81d9ed7e63a79f6824cfed148b666bf7678f45e3",
    "pipeline": "master"
  },
  {
    "project": "pass-refreshid",
    "component": "enforcerd",
    "pr-id": "1769",
    "commit-sha": "8ed276ce885578c4eddd7183e1e8a1f7955c835f",
    "pipeline": "master"
  },
  {
    "project": "pass-refreshid",
    "component": "backend",
    "pr-id": "771",
    "commit-sha": "d34a308c234c4ff96098fa87d895d41da1a2b986",
    "pipeline": "master"
  }
]
```

**Comment by sibicramesh** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "pass-refreshid",
    "component": "backend",
    "pr-id": "771",
    "commit-sha": "3a1e5477871c6b7c7cf7da1eafeca1f7df4af394",
    "pipeline": "master"
  },
  {
    "project": "pass-refreshid",
    "component": "gaia",
    "pr-id": "509",
    "commit-sha": "81d9ed7e63a79f6824cfed148b666bf7678f45e3",
    "pipeline": "master"
  },
  {
    "project": "pass-refreshid",
    "component": "enforcerd",
    "pr-id": "1769",
    "commit-sha": "8ed276ce885578c4eddd7183e1e8a1f7955c835f",
    "pipeline": "master"
  }
]
```

**Pull request merged**